### PR TITLE
Rearranged Contact Page

### PR DESCRIPTION
**Pull request by sean-william**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -473,15 +474,39 @@ <h4>Diana Pertersen</h4>
...
+          <h3 class="section-subheading text-white-50">If you want to know more about our products, features, pricing or anything at all, we are here to answer your questions</h3>
         </div>
       </div>
       <div class="row">
         <div class="col-lg-12">
-          <form id="contactForm" name="sentMessage" novalidate="novalidate">
             <div class="row">
               <div class="col-md-6">
+			  <p class="h2 text-white">Office Locations</p>
+			  </br>
```
Comment (josephting) [inline index.html:486]: Don't use `<br>`.

Use [Bootstrap Spacing](https://getbootstrap.com/docs/4.2/utilities/spacing/) instead?
Maybe `mb-3` or `mb-4`?

File: index.html [hunk 2]
```diff
@@ -473,15 +474,39 @@ <h4>Diana Pertersen</h4>
...
+			    </br>
+			  <p class="h4 text-white">Hanoi</P>
+			  <p class="h6 text-white-50">No. 6, Alley 39,</p>
+			  <p class="h6 text-white-50">Pham Than Duat Street,</p>
+			  <p class="h6 text-white-50">Mai Dich Ward,</p>
+			  <p class="h6 text-white-50">Cau Giay District,</p>
+			  <p class="h6 text-white-50">Hanoi, Vietnam.</p>
+			  </br>
+			  <p class="h6 text-white-50">+(84)24 6282 2821</p>
+			  </br>
```
Comment (josephting) [inline index.html:506]: Why end with `<br>`?

**Review by josephting — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (sean-william) [reply index.html:486]: okay. Got it!

File: index.html (hunk 2 above)
Comment (sean-william) [reply index.html:506]: copy pasted from the top section haha

**Pull request merged**